eli5/formatters/text_helpers.py

```diff
@@ -0,0 +1,81 @@
+from collections import Counter
+from typing import List
+
+import numpy as np
+
+from eli5.base import TargetExplanation, WeightedSpans, DocWeightedSpans
+from eli5.base_utils import attrs
+from .utils import max_or_0
+
+
+def get_char_weights(doc_weighted_spans, preserve_density=None):
+    # type: (DocWeightedSpans, bool) -> np.ndarray
+    """ Return character weights for a text document with highlighted features.
+    If preserve_density is True, then color for longer fragments will be
+    less intensive than for shorter fragments, so that "sum" of intensities
+    will correspond to feature weight.
+    If preserve_density is None, then it's value is chosen depending on
+    analyzer kind: it is preserved for "char" and "char_wb" analyzers,
```

Comment:
It'd be good for formatters not to be aware of scikit-learn. Maybe it is just a matter of documenting things differently.